src/modules/transactions/hooks/signature/useSignTransaction.ts

```diff
@@ -27,19 +27,20 @@ interface IUseSignTransactionProps {
   transactionList: IUseTransactionList;
   pendingSignerTransactionsRefetch: () => void;
   homeTransactionsRefetch: () => void;
+  pendingTransactions: IPendingTransactionsRecord;
 }
 
 const useSignTransaction = ({
   transactionList,
   pendingSignerTransactionsRefetch,
   homeTransactionsRefetch,
+  pendingTransactions,
 }: IUseSignTransactionProps) => {
   const {
-    pendingTransactions,
     request: { refetch: transactionsPageRefetch },
   } = transactionList;
   const [selectedTransaction, setSelectedTransaction] =
-    useState<ITransaction>();
+    useState<Partial<ITransaction>>();
```

Comment:
replace:    `useState<Partial<ITransaction>>();` -> `useState<Pick<Transaction, 'name', 'id', ...others>>()`

Reply:
I got an interface for that: 

`export interface IPendingTransactionDetails {
  status: TransactionStatus;
  hash: string;
  id: string;
  predicateId: string;
  resume: ITransactionResume;
}`